### ISSUE TITLE
Cannot build on Windows 10

### ISSUE THREAD

**Issue by Candunc** [sgrif/mysqlclient-sys#22]
I've tried every combination of MySQL connector and MariaDB connector and none will build. ```vcpkg install libmysql:x64-windows``` no longer provides a mysqlclient.lib file, and ```vcpkg install libmariadb:x64-windows``` and renaming to libmysql doesn't work either. Library files from MySQL Server 8.0 and 5.7 result in the same errors. 

> error: linking with `link.exe` failed: exit code: 1120
>   |
>   = note: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX64\\x64\\link.exe" "/NOLOGO" "/NXCOMPAT" "/LIBPATH:C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.12ru3ld426f5qxjf.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.14i6hm2h356tbd8i.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.158196a816knso10.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.16t32wvqfjdmt3r0.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.18e70bm03qzb7qn.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.19zaeza6owzk7b21.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1c0iu4d117c2d3g4.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1c9nzr9x8gbt90ud.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1ce6zmrsue33m56x.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1j9o2s4ijtaw2u8v.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1sz3ojpjo6uz226r.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1ulxcoc7fmx2dxd5.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1vlhofe7p3xaouq4.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1wb50r883pfbez0d.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1whrth6kff7lybqr.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1xwmdqu4psj93zov.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.1ydw11phodtibm06.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.25oma8s13awtw4yo.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.267sl21wfhk05z6p.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.28oym4vsxh21ov6t.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2a0br25z2hsgm7hk.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2a8h699xxkk07yqt.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2apx1l1ekkvciucs.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2cvgte407hwvyrg2.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2dmzf8cnfpc0p18i.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2dow32ox9z4ykgfp.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2ehy255si9sdewan.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2lc3u653ay52co44.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2nofraq825gmqhoi.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2odev6pfagajbiza.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2pdpa3a51jzoy9ka.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2pq62jly41334uq2.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2umg9bwdyzuf79oh.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2uqigkcwfo76hg4y.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.2yx3o94s7tdwcqip.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.353yz1d0cmgcpgtk.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.36j510zahf9lmw4x.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3b364vw3gotmx3fg.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3dq19snu8o0s14g2.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3eeut5i7af73luox.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3evqd9s5of5sl134.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3ihq8wc2g71crazt.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3jka9890f7nn0xd0.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3mvky90ptaf4lvwq.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3pz3jgyomsus1eda.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3svmh3p0n27n51oj.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3uajt4m01ul2jcyv.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.3vwwybfs50cnhy9q.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.430w5ev078lsu63z.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.44y48e54x7u3fc3n.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4a17gqz0uljmqglz.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4d3k6euk18v2c49a.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4eav14ujxsz60gc5.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4nvns7tbapmanmc2.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4qjiclb9isajo2xi.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4vhnn39rtgmumn96.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4wl8qwyv3vtnayww.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.4ywh3xydcnltnagd.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.50rqve5qhvoa0314.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.52wg18jfjdqz4nbw.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.543qap3kp09m4vs.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.54zqvrdbohvdj072.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.56sohp0lekckyi63.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.59o6rjau1k9v48w2.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.5aqsubzf588ai29v.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.5b0g9zebbvhrsyh5.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.5e4gyhkpna2xg8jr.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.5eelx6y13mqi4vj3.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.6md4c6ynzvx7gz6.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.cxw8wpvs4y0vpdw.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.dmd9pgeo4emx2fz.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.geoi5pquz9t4mpg.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.i00jioo5wrb0haf.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.it7uh9illfywvz5.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.ld320n5698ipms6.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.mr2m9i7tnx9qhct.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.n1esf7eslmqg0jn.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.ryfqct0348nz4l.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.rzhesyw5tvw4mho.rcgu.o" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.srwt7acp0zm1v5r.rcgu.o" "/OUT:D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.exe" "D:\\Repositories\\test-server\\target\\debug\\deps\\test_server.19xrjbz46z4ow3z7.rcgu.o" "/OPT:REF,NOICF" "/DEBUG" "/NATVIS:C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\intrinsic.natvis" "/NATVIS:C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\liballoc.natvis" "/NATVIS:C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libcore.natvis" "/NATVIS:C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\etc\\libstd.natvis" "/LIBPATH:D:\\Repositories\\test-server\\target\\debug\\deps" "/LIBPATH:C:\\Program Files\\MySQL\\MySQL Server 8.0\\lib" "/LIBPATH:D:\\Repositories\\test-server\\target\\debug\\build\\ring-bc1507183d48bbe3\\out" "/LIBPATH:C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib" "D:\\Repositories\\test-server\\target\\debug\\deps\\librocket_contrib-75b60937cdba6afe.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libserde_json-ba338291c254485e.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libryu-8f90530b2e67412d.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libitoa-583549cb6fd9c400.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\librocket-f3baeb146003863b.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libatty-48253c35cfaab544.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libbase64-2b5c2995310f7b53.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libmemchr-60db776ff2bddf5b.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libtoml-c9e114c0df0dd8cc.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libyansi-c0a335bd87df385f.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\librocket_http-786631d5c2c0956f.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libhyper-c7bde80101ce2201.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libmime-dd12204d543f5469.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\liblog-5814bb45ec4d49e4.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\liblanguage_tags-1972bae6d65224de.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libtypeable-85d254938ba0fc3d.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libtraitobject-c6c662449c7540b6.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libnum_cpus-bd7602212536bb3a.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libhttparse-5041999f61e86084.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libunicase-0ccfa50d3feef342.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libunicode_xid-eb65be163b8bf12a.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libstate-2518153af98831dd.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libindexmap-6b90e7039d39f7d7.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libcookie-4c4ca3fd07c28400.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libbase64-8e61546fbebdd492.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libsafemem-327ad297a48d8cbc.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libring-14454dfbe646f5ee.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libuntrusted-99652e0b7bcadba3.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libtime-2ca57436aca4d57e.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\liburl-063071d7d0fa1248.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libidna-51e4deb52ed6b2b4.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libpercent_encoding-a003777f6a9c6e94.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libpear-d39b879992d7b87c.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libserde-68a8e1e86c055e24.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libdiesel-7e96e98e730e8f5f.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\liburl-eca4821d83412b81.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libidna-ea13cb77d3810ca6.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libunicode_normalization-2e0c24246bce4a43.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libunicode_bidi-6df7a071799149eb.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libmatches-95cebda6bb29887d.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libpercent_encoding-7a8174fb11529e5a.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libmysqlclient_sys-9976c8667e720a94.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libr2d2-e284e7874f6ebef0.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libscheduled_thread_pool-719e78f6724bd80a.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libparking_lot-aa40c30bd67806ae.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libparking_lot_core-d1efab608c9efe8a.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libwinapi-0b230fa248d80fde.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libsmallvec-6885c777327717bc.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\liblock_api-07bee408729fca1d.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libscopeguard-3d379cff07f787f1.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\liblog-b6352f086f1083a8.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libcfg_if-406d7160e886c022.rlib" "D:\\Repositories\\test-server\\target\\debug\\deps\\libbyteorder-64eae96a6c39cac1.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libstd-fae0adc6a4bf13fc.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libpanic_unwind-a7b685aa806e9051.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libhashbrown-959d4337805b4823.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_alloc-793090c0698ed48d.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libbacktrace-0b2ee9875ceef572.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_demangle-512b5012620b8d24.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libunwind-fd322e9a5f11a20b.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcfg_if-ddd2bba294770bdf.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liblibc-9eacc70b3ac746bf.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\liballoc-2605391ad035cd8d.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\librustc_std_workspace_core-e30d182074a5a412.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcore-83168dc81ba47b29.rlib" "C:\\Users\\DuncB\\.rustup\\toolchains\\nightly-x86_64-pc-windows-msvc\\lib\\rustlib\\x86_64-pc-windows-msvc\\lib\\libcompiler_builtins-2c5b191d679853a8.rlib" "advapi32.lib" "advapi32.lib" "cfgmgr32.lib" "fwpuclnt.lib" "gdi32.lib" "kernel32.lib" "msimg32.lib" "ntdll.lib" "opengl32.lib" "synchronization.lib" "user32.lib" "winspool.lib" "ws2_32.lib" "advapi32.lib" "ws2_32.lib" "userenv.lib" "msvcrt.lib"
>   = note:    Creating library D:\Repositories\test-server\target\debug\deps\test_server.lib and object D:\Repositories\test-server\target\debug\deps\test_server.exp
>           libmysqlclient_sys-9976c8667e720a94.rlib(handshake.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_lib.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(ctype-uca.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_winfile.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_file.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_default.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(vio.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(sha2_password_common.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(handshake_client.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(client.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(charset.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(client_plugin.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(compression.obj) : error LNK2001: unresolved external symbol __CxxFrameHandler4
>           libmysqlclient_sys-9976c8667e720a94.rlib(ctype-uca.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_fopen.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_winfile.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(vio.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_open.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(sha2_password_common.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(handshake_client.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(client.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(charset.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(compression.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           libmysqlclient_sys-9976c8667e720a94.rlib(my_default.obj) : error LNK2001: unresolved external symbol __GSHandlerCheck_EH4
>           D:\Repositories\test-server\target\debug\deps\test_server.exe : fatal error LNK1120: 2 unresolved externals
> 
> 
> error: aborting due to previous error
> 
> error: could not compile `test-server`.
> 
> To learn more, run the command again with --verbose.

**Comment by zzhengzhuo** [comment]
I met the same question.Maybe you try adding '--target x86_64-pc-windows-msvc'.

**Comment by mikeoaklet** [comment]
Same problem here.

**Comment by WeaponMan** [comment]
I did some digging after it started bugging me that my project doesn't compile on windows.
I did figure out that in order to compile diesel and link it succesfully you need to install `libmysql:x64-windows-static-md` instead of `libmysql:x64-windows`.

**Comment by WeaponMan** [comment]
Make sure you have `VCPKG_ROOT` in the env variables poitining to the vckpg instalation.
I did firstly install `libmysql:x64-windows`, clean rebuild of the project got me same error message.
Followed by installing `libmysql:x64-windows-static` same error message.
Then I read this https://docs.rs/vcpkg/0.2.10/vcpkg/ and installed `libmysql:x64-windows-static-md` version and it started building.

Even before all of that I tried the mysql C connetor with added the library to build.rs got a linking error every time so that doesnt work for sure.

PS: this is a nigthmare, on debian I installed one package and it was building
